ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -48,19 +48,31 @@ tests:
   steps:
     cluster_profile: packet
     env:
-      ASSISTED_CONFIG: |
-        OPENSHIFT_VERSION=""
       TEST_SUITE: minimal
     workflow: baremetalds-assisted-install
 - as: e2e-metal-assisted-periodic
   cron: '@daily'
   steps:
     cluster_profile: packet
     env:
-      ASSISTED_CONFIG: |
-        OPENSHIFT_VERSION=""
       TEST_SUITE: full
     workflow: baremetalds-assisted
+- as: e2e-metal-assisted-multi-version
+  steps:
+    cluster_profile: packet
+    env:
+      ASSISTED_CONFIG: |
+        OPENSHIFT_VERSION=""
+      TEST_SUITE: minimal
+    workflow: baremetalds-assisted-install
+- as: e2e-metal-assisted-multi-version-periodic
```

Comment:
where's the periodic job defined? I only see the config

Reply:
Good question

Reply:
I mistakenly forgot the `cron` stanza. Fixed. :)